src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,26 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final String NATIONALITY = "Ukrainian";
+    private static final int MIN_AGE = 35;
+    private static final int YEARS_OF_LIVING = 10;
+    private static final int FIRST_INDEX_OF_ARRAY = 0;
+    private static final int SECOND_INDEX_OF_ARRAY = 1;
```

Comment:
let's try to name variables according to the business meaning, not just duplicate the value in the name.
```suggestion
    private static final int START_PERIOD = 0;
    private static final int END_PERIOD = 1;
```